url9e019a0ee9_elfrasco_design_principles_tgz-pJ8-ar_com_epidataconsulting_principles_ioc_before_SavePredictiveEngineProjectServiceJ8/src/ar/com/epidataconsulting/principles/ocp/before/EmployeeMockBuilder.java

```diff
@@ -0,0 +1,62 @@
+package ar.com.epidataconsulting.principles.ocp.before;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+public class EmployeeMockBuilder {
+
+	public static Employee createFlorenciaCastagnino() {
+		Employee employee = new Employee();
+		employee.setName("Florencia");
+		employee.setLastName("Castagnino");
+		employee.setDateOfBirth(new GregorianCalendar(1980, Calendar.JANUARY, 15).getTime());
+		employee.setFileNumber("123");
+		return employee;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 9:14, 28:33, 37:42, 46:51, 55:60.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.